torch_xla/utils/utils.py

```diff
@@ -8,6 +8,42 @@
 import sys
 import tempfile
 import time
+import json
+from collections import defaultdict
+
+import torch_xla.utils.gcsfs as gcsfs
+
+
+class CheckpointTagger(object):
```

Comment:
Do not stuff utils.py. Make a separate file.

Reply:
done.